src/cryptoadvance/specter/services/extension_gen.py

```diff
@@ -46,13 +48,20 @@ def __init__(
         self.base_path = base_path
         self.org = ext_org
         self.id = ext_id
+        self.encrypted_userdata = encrypted_userdata
+        self.tag = tag
```

Comment:
You introduce here `self.tag` which is a better name than `self.branch`. Can you remove the `self.branch` then and adjust the usage? E.g. the `GithubUrlLoader` currently has a parameter called "branch" but it should be called `tag` (and initialized with your `self.tag`.